### PR TITLE
chore(agent): show sample duration in endpoint, cleanup, reduce required min funds slightly

### PR DESCRIPTION
**Pull request by istae**
### Checklist

- [x] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [ ] I have filled out the description and linked the related issues.

### CODE REVIEW

File: pkg/storageincentives/agent.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ const (
 	DefaultBlocksPerPhase = DefaultBlocksPerRound / 4
 
 	// min # of transactions our wallet should be able to cover
-	minTxCountToCover = 25
+	minTxCountToCover = 15
```
Comment (ldeffenb) [inline pkg/storageincentives/agent.go:39]: What does this change have to do with the subject of the PR?

File: pkg/storageincentives/agent.go [hunk 2]
```diff
@@ -129,8 +129,6 @@ func (a *Agent) start(blockTime time.Duration, blocksPerRound, blocksPerPhase ui
 			a.logger.Error(err, "phase failed", "phase", phase, "round", round)
 		} else if isPhasePlayed {
 			a.logger.Info("phase played", "phase", phase, "round", round)
-		} else {
```
Comment (ldeffenb) [inline pkg/storageincentives/agent.go:132]: Was this informative log removed on purpose?

File: pkg/storageincentives/agent.go [hunk 3]
```diff
@@ -402,20 +401,21 @@ func (a *Agent) handleSample(ctx context.Context, round uint64) (bool, error) {
 	}
 
 	if !hasFunds {
-		a.logger.Info("insufficient funds to participate in next round", "round", round)
+		a.logger.Info("insufficient funds to play in next round", "round", round)
```
Comment (ldeffenb) [inline pkg/storageincentives/agent.go:404]: Beginning to think this PR was started from the wrong base version?

**Review by ldeffenb — CHANGES_REQUESTED** [review]
More changes than this PR title would seem to address.

File: pkg/storageincentives/agent.go (hunk 2 above)
Comment (istae) [reply pkg/storageincentives/agent.go:132]: yeah, it's unnecessary to log that the node is not playing the round in every phase

**Review by mrekucci — APPROVED** [review]

**Pull request merged**